src/transformers/models/paligemma/processing_paligemma.py

```diff
@@ -74,19 +74,45 @@ def build_string_from_input(prompt, bos_token, image_seq_len, image_token):
     """
     return f"{image_token * image_seq_len}{bos_token}{prompt}\n"
 
+
+class PaliGemmaTextKwargs(TextKwargs):
+    suffix: Optional[Union[TextInput, PreTokenizedInput, List[TextInput], List[PreTokenizedInput]]] = None
+
+
+class PaliGemmaImagesKwargs(ImagesKwargs):
+    do_convert_rgb: Optional[bool] = None
+    do_thumbnail: Optional[bool] = None
+    do_align_long_axis: Optional[bool] = None
+    do_rescale: Optional[bool] = None
+
+
 class PaliGemmaProcessorKwargs(ProcessingKwargs, total=False):
+    text_kwargs: PaliGemmaTextKwargs
+    image_kwargs: PaliGemmaImagesKwargs
     _defaults = {
         "text_kwargs": {
-            "tokenize_newline_separately": True,
-            "suffix": None,
+            "tokenize_newline_separately": True,  # Not Available in Default
+            "suffix": None,  # Not Available in Default
+            "padding": False,
+            "truncation": None,
+            "max_length": None,
         },
         "image_kwargs": {
-            "do_convert_rgb": None,
-            "do_thumbnail": None,
-            "do_align_long_axis":  None,
+            "do_resize": None,
+            "do_normalize": None,
+            "image_mean": None,
+            "image_std": None,
+            "data_format": "channels_first",
+            "input_data_format": None,
+            "resample": None,
+            "do_convert_rgb": None,  # Not Available in Default
+            "do_thumbnail": None,  # Not Available in Default
+            "do_align_long_axis": None,  # Not Available in Default
+            "do_rescale": None,
```

Comment:
nb: `None` defaults don't need to be included in `<MyModel>ProcessorKwargs` as it's the normal default!